pkg/collect/logs_test.go

```diff
@@ -77,3 +79,62 @@ func Test_setLogLimits(t *testing.T) {
 		})
 	}
 }
+
+func Test_savePodLogs(t *testing.T) {
+	tests := []struct {
+		name              string
+		withContainerName bool
+		want              CollectorResult
+	}{
+		{
+			name:              "with container name",
```

Comment:
Could we improve the description such as should fail ... or should pass when ...

Reply:
The BDD styly naming of the tests does not feel quite right here IMHO

```
=== RUN   Test_savePodLogs
=== RUN   Test_savePodLogs/should_have_a_container_name_in_path_when_container_name_is_defined
=== RUN   Test_savePodLogs/without_container_name
--- PASS: Test_savePodLogs (0.00s)
```

Reply:
Hun, that is a nit and not a blocker at all.

However, 

`=== RUN   Test_savePodLogs/without_container_name` Should pass or fail? What is the expected behaviour?

About BDD style, see: https://dev.to/smyrman/test-with-expect-a-bdd-style-go-naming-pattern-5eh5#:~:text=GWT%2C%20or%20%22Given%2DWhen,you%20start%20the%20detailed%20implementation.

Example
```
TestSum/With_non-empty_int_slice/Expect_no_error
TestSum/With_non-empty_int_slice/Expect_correct_sum
TestSum/With_non-empty_int_slice/Expect_input_is_unchanged
TestSum/With_empty_int_slice/Expect_no_error
TestSum/With_empty_int_slice/Expect_zero
TestSum/With_empty_int_slice/Expect_input_is_unchanged
```

So, should we not expected something like:

```
=== RUN   Test_savePodLogs/When_container_name_is_defined_expected_name_in_the_path
=== RUN   Test_savePodLogs/When_container_name_is_undefined_expected_no_error 
--- PASS: Test_savePodLogs (0.00s)
```